.github/dependabot.yml

```diff
@@ -0,0 +1,6 @@
+version: 2
+updates:
+- package-ecosystem: "github-actions"
+  directory: "/"
+  schedule:
+    interval: "weekly"
```

Comment:
weekly means once a week the bot will create PRs to update all charts that depend on a child that was updated right?

Reply:
This PR is for github actions only,  dependabot doesn't support helm charts yet. I am working on an alternative for that. Will create a different PR for it. 


This PR will create a PR once a week if there a new version for any of the github actions we use. 
eg: https://github.com/maniteja6799/chart-actions-demo/pull/12